libpod/container.go

```diff
@@ -174,6 +174,7 @@ type ContainerConfig struct {
 	// TODO consider breaking these subsections up into smaller structs
 
 	// Storage Config
+	IDMappings storage.IDMappingOptions
```

Comment:
Needs JSON tag and a comment explaining what this field does

Reply:
Also, can you make an accessor function that returns this, so people can see what mappings a container has through our API?